src/test/org/pantsbuild/jarjar/integration/IntegrationTestBase.java

```diff
@@ -0,0 +1,262 @@
+package org.pantsbuild.jarjar.integration;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import junit.framework.TestCase;
+import org.junit.rules.TemporaryFolder;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Base class for jarjar integration tests. Provides common utilities.
+ */
+public abstract class IntegrationTestBase extends TestCase {
```

Comment:
I could; this just seemed a little more thorough since it also exercises how options are input (jarjar uses java system properties for some reason) etc.